rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -4247,6 +4290,8 @@ enum MessageBuilder {
     Request(CanisterId, CanisterId),
     // `(respondent, originator)`
     Response(CanisterId, CanisterId),
+    // `(respondent, originator, deadline)`
+    BestEffortResponse(CanisterId, CanisterId, CoarseTime),
```

Comment:
Nit: wouldn't it be cleaner to just have named fields instead?

(Just an idea, feel free to drop it if it would require too many changes.)